pkg/build/templates/modify-raw-image.sh.tpl

```diff
@@ -13,6 +13,17 @@ set -euo pipefail
 #
 # Guestfish Command Documentation: https://libguestfs.org/guestfish.1.html
 
+# In x86_64, the default root partition is the third partition
+ROOT_PART=/dev/sda3
+
+# Make the necessarry adaptations for aarch64
+if [[ $(uname -m) == "aarch64" ]]; then
```

Comment:
I think we should use the variable that’s set to the arch in the image definition file and then template it in as it allows us to remove this if check and just rely on a single variable which is a little bit safer.

Reply:
I vote for doing this too - we should be able to simplify the script and move everything to Go code.